locales/bn_BD/LC_MESSAGES/duckduckgo.po

```diff
@@ -199,7 +199,7 @@ msgid "Add DuckDuckGo as a search engine"
 msgstr "সার্চ ইঞ্জিন হিসেবে DuckDuckGo-কে যোগ করুন"
 
 msgid "Add DuckDuckGo to %s"
-msgstr ""
+msgstr "%s -এ ডাকডাকগো যোগ করুন"
```

Comment:
We prefer not to translate our brand name. I'm adding back the brand name. Thanks!

```suggestion
msgstr "%s -এ DuckDuckGo যোগ করুন"
```